.github/workflows/prbot.yml

```
@@ -104,7 +104,13 @@ jobs:
       - run: npm run build
       - name: Identify E2E Test Files to run
         id: determine-tests
-        run: node ./scripts/ci/determine-tests.mjs testsToRun shardIndex shardTotal
+        run: |
+
+          if [ "${{ github.event_name }}" == "merge_group" ]; then
+            node ./scripts/ci/determine-tests.mjs testsToRun shardIndex shardTotal true
+          else
+            node ./scripts/ci/determine-tests.mjs testsToRun shardIndex shardTotal
+          fi
```

I think this putting `false` would be clearer with the combination of `const runAllTests = process.argv[5]`
````suggestion
        run: |

          if [ "${{ github.event_name }}" == "merge_group" ]; then
            node ./scripts/ci/determine-tests.mjs testsToRun shardIndex shardTotal true
          else
            node ./scripts/ci/determine-tests.mjs testsToRun shardIndex shardTotal false
          fi
```